content/en/blog/2022-04-04-march-update/index.md

```diff
@@ -0,0 +1,673 @@
+provides an intuitive way to manage, troubleshoot and operate your
+Kubernetes environment following the GitOps operating model,
+accelerating your development lifecycle and simplifying your continuous
+delivery pipelines.
+
+Weaveworks GitOps Extension integrates with [Kubernetes
+Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools),
+[kubectl](https://kubernetes.io/docs/reference/kubectl/overview/)
+and [flux](/) for a consolidated and tightly integrated user experience.
+
+{{< tweet user=GeertBaeke id=1504757889977798675 >}}
+
+{{< youtube -07emkW8eiM >}}
+
+🚧 This extension is under active development and currently available as
+an alpha product.
+
+#### Flux Subsystem for Argo
+
+[FSA (aka Flamingo)](https://github.com/flux-subsystem-argo/flamingo)
+is the Flux Subsystem for Argo. Its container image can be used as a drop-in
+replacement for the equivalent ArgoCD version to visualise, and manage
+Flux workloads, alongside ArgoCD.
+
+How does it work?
+
+![Flux Subsystem for Argo](fsa.png)
+
+🚧 This project is currently available as a technology preview.
+
+#### Terraform-controller
+
+In some of our last issues we already reported about the
+[terraform-controller hitting the
+streets](/blog/2022/01/january-update/#community-project-terraform-controller-for-flux).
+It's a [Flux controller which reconciles
+Terraform](https://github.com/chanwit/tf-controller)
+resources in the GitOps way. We received a short report from the team
+regarding their achievements of the first quarter of the year:
+
+- TF-controller
+  [v0.9.3](https://docs.google.com/document/u/0/d/1Z6yPN9-bWeVGpMrBxXJ3NBTBYZowJ3R93wKHEVmBJ1A/edit)
+  is considered the most stable release to date.
+- We reached 200 stars on GitHub, now at 211.
+- It\'s been 45 releases so far.
+- We re-factored it to the Controller/Runner architecture.
+- Standing on the shoulders of our giants (Flux), we successfully
+  implemented the multi-tenancy feature in 2 months.
+- We cleared all Q1
+  [roadmap](https://github.com/weaveworks/tf-controller#roadmap)
+  with 68.2% test coverage.
+- We started seeing its adoption in public, from [a Helm Controller
+  user](https://github.com/fluxcd/helm-controller/issues/448),
+  for example.
+- We got its [first promo
+  video](https://www.linkedin.com/embed/feed/update/urn:li:ugcPost:6913743507681656832).
+- Chanwit Kaewkasi, Piaras Hoban and Tom Huang are the core team
+  around it now!
+
+#### Weave GitOps Core
+
+The team around Weave GitOps has been busy and would love to hear your
+feedback. If you haven't heard about it just yet, its Github says:
+
+> [Weave GitOps](https://github.com/weaveworks/weave-gitops) enables an effective
+> GitOps workflow for continuous delivery of applications into Kubernetes
+> clusters. It is based on CNCF Flux, a leading GitOps engine.
+
+Particularly the last sentence the Flux community loved reading of course.
```

Comment:
```suggestion
The Flux community particularly loved the last sentence.
```

Reply:
Might want to add exclamation mark, or emoji per your preferred style. :slightly_smiling_face: